lib/Dialect/TritonGPU/Transforms/Coalesce.cpp

```diff
@@ -38,7 +38,7 @@ struct CoalescePass : public impl::TritonGPUCoalesceBase<CoalescePass> {
     });
 
     auto contiguity = axisInfoAnalysis.getAxisInfo(ptr)->getContiguity();
-    SmallVector<unsigned> order = argSort(contiguity);
```

Comment:
is `argSort` helper dead? Can we remove it?

Reply:
yes it's not used anymore, removed.